src/components/list-card.html

```diff
@@ -1,15 +1,21 @@
 <div class="card">
-  <figure>
-    <img src="{{ img }}" alt="pl" class="course-img"/>
+  <figure class="course-layout">
+    <img src="{{ img }}" alt="course's image" class="course-img"/>
   </figure>
-  <div class="texts">
-    <p>{{ content }}</p>
-    <div class="course-symbol">
+  <div class="texts ">
+    <table class="course-tabble">
+      <tr>
+        <th>
+           <p class="course-layout">{{ content }}</p>
+        </th>
+      </tr>
+    </table>
+    <div class="course-symbol ">
       <b>4.0</b>
       <div class="Stars" style="--rating: 4;" aria-label="Rating of this product is 4 out of 5."></div>
```

Comment:
i think u should defind variable for rating 
example: `style="--rating: {{rating}}` and reused it for flexibility, not fixed rating